docker_mcp/core/transfer/archive.py

```
@@ -3,11 +3,13 @@
 import asyncio
 import subprocess
 from datetime import datetime
+from pathlib import Path
 from typing import Any
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)